src/lambdas/initiateAthenaQuery/handler.test.ts

```diff
@@ -174,4 +185,18 @@ describe('initiate athena query handler', () => {
     )
     expect(mockUpdateQueryByZendeskId).not.toHaveBeenCalled()
   })
+
+  it('checks that the lambda exits early and does not run an athena query as it is a manual query', async () => {
+    const testZendeskId = testManualAthenaQueryEvent.Records[0].body
+    mockPublishToSNS.mockResolvedValue('messageID')
+    await handler(testManualAthenaQueryEvent, mockLambdaContext)
+    expect(mockPublishToSNS).toHaveBeenCalledWith(
+      'arn:aws:sns:eu-west-2:123456789012:email-to-slack-topic',
+      `Retrieved data for zendeskID: ${testZendeskId}`
+    )
+    expect(logger.info).toHaveBeenCalledWith(
+      'Manual query detected, no need to run athena query'
+    )
+    expect(confirmAthenaTable).not.toHaveBeenCalled()
```

Comment:
If you're leaving the other dependencies in place, I think I'd like to see some assertions that they're also not called. At a minimum, you want to make sure `startQueryExecution` isn't called.

Reply:
added a wrapper